### DISCUSSION TITLE
Range fields can not be set as required

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#484] [_Discarded pitches]
### Current state
Some fields in interest forms are range fields with two inputs, the lower and upper limit (“from” and “to”).

An example is “Kaufpreis” (purches price) in https://testinstanz-fal.onofficeweb.com/interessentenformular/

When such a field is marked as “required” in the WordPress backend, the form can not be sent. That field is then always considered missing, even when the user actually has filled in the values as required.

The cause of this is that the field is transformed to a range by adding “__von” and “__bis” to its field name (e.g. “kaufpreis__von”). The check for missing fields does not consider this and thus wrongly determines that the original field (e.g. “kaufpreis”) is missing.

### Desired state
The check for missing fields needs to consider the “from” and “to” fields, so the lower and upper limits can be filled in, even when the field is marked as required.

**Comment by fredericalpers** [comment]
@dai-eastgate Please investigate and let me know how long this approx. will take.

**Reply by dai-eastgate** [reply]
@fredericalpers I need 4 days to fix and test it.
